test/helpers/utils.go

```diff
@@ -147,6 +150,61 @@ func WithTimeoutErr(ctx context.Context, f func() (bool, error), freq time.Durat
 	}
 }
 
+// InstallExampleCilium uses Cilium Kubernetes example from the repo,
+// changes the etcd parameter and installs the stable tag from docker-hub
+func InstallExampleCilium(kubectl *Kubectl) {
+
+	var path = "../examples/kubernetes/cilium.yaml"
```

Comment:
Does this means this will be the only place I've to change to `../examples/kubernetes/K8S_VERSION/cilium.yaml` in order to merge #3125?

Reply:
yes!